proto/raft_serverpb.proto

```diff
@@ -252,6 +252,7 @@ enum ExtraMessageType {
     MsgGcPeerResponse = 12;
     MsgFlushMemtable = 13;
     MsgRefreshBuckets = 14;
+    MsgAckCommittedIndex = 15;
```

Comment:
I think use two message types for the request and resp is better.

Reply:
+1